.baseline/checkstyle/checkstyle.xml

```diff
@@ -414,6 +414,11 @@
             <property name="format" value="@Test\(.*expected.*\)"/>
             <property name="message" value="Prefer using Assertions.assertThatThrownBy(...).isInstanceOf(...) instead."/>
         </module>
+        <module name="RegexpSinglelineJava">
+            <property name="format" value="^\s*assert\s+"/>
```

Comment:
current pattern needs some improvement, since it also matches `fooassert` (shouldn't) and doesn't match `assert(true);` (should).
instead of `\s*` / `\s+`, check it's a word boundary: `\b` (`\bassert\b`)


I thought there should be a dedicated way to do this, without having to use regular expressions.
in fact, error-prone claims to do this: https://errorprone.info/bugpattern/UseCorrectAssertInTests . why it didn't work for us?

Reply:
can you give https://checkstyle.sourceforge.io/checks/coding/illegaltoken.html#IllegalToken with LITERAL_ASSERT token a try?

Reply:
I don't think we want to match code that doesn't compile or isn't properly formatted. `fooassert´ isn't a valid statement and `assert(true)` would be flagged as not properly formatted code by spotless, thus resulting in `assert (true)`.

Reply:
@findepi `LITERAL_ASSERT` does the trick and is much better than regex matching, thanks